docs/stylesheets/extra.css

```diff
@@ -0,0 +1,42 @@
+small.subtitle {
+  display: block;
+  margin-top: -1.2em;
+}
+
+dl.schema-column-list {
+  font-size: .64rem;
+  border: .05rem solid var(--md-typeset-table-color);
+  border-radius: .1rem;
+  border-bottom: none;
+}
+
+dl.schema-column-list > div {
+  border-bottom: .05rem solid var(--md-typeset-table-color);
+  padding: .7720588235em 1.1764705882em;
```

Comment:
question: Where do these numbers come from? They look like they've been calculated.

Reply:
I just used DevTools to grab styles from existing bits of the theme. In this particular case, I wanted the definition list to have the same styling as a table so I just found an example table and borrowed the styles from there.